### ISSUE TITLE
Consolidate "white button with blue text" style

### ISSUE THREAD

**Issue by danacotoran** [alphagov/govuk_publishing_components#2238]
The components gem currently has several components which use a similarly styled button, for instance: 
https://components.publishing.service.gov.uk/component-guide/govspeak/charts 
https://components.publishing.service.gov.uk/component-guide/subscription_links/with_copyable_feed_link 
https://components.publishing.service.gov.uk/component-guide/print_link

This style of button is not currently standardised, and each of these individual components has custom styling to achieve more or less the same result. 

Investigate if we can consolidate these styles into a new variation of the button component, in order to reduce repetition in our CSS.

**Comment by chris-gds** [comment]
Agreed, in theory I would perhaps say these qualify as [secondary buttons](https://design-system.service.gov.uk/components/button/)

![image](https://user-images.githubusercontent.com/71266765/132886093-7054d3ba-40b7-443f-a887-97f10da3d54f.png)

**Comment by andysellick** [comment]
Charts button now uses a standard secondary style button.

**Comment by andysellick** [comment]
The remaining buttons referred to by this issue are:

[button in subscription links](https://components.publishing.service.gov.uk/component-guide/subscription_links/with_copyable_feed_link) | [print link button](https://components.publishing.service.gov.uk/component-guide/print_link)
---- | -----
<img width="415" height="81" alt="Image" src="https://github.com/user-attachments/assets/b1f491fb-5cad-4c37-a23d-145ede0703b7" /> | <img width="193" height="72" alt="Image" src="https://github.com/user-attachments/assets/8d67cab3-94f9-4a00-90c2-6eb0c8742d77" />

They're both similar (1px border, white background, contain SVG) but different (subs links button is bold, SVGs included in button text in first, background image in CSS in second).

Options are:

## Changing them to use a different button style, such as secondary button
Not really an option without causing visual inconsistencies. The print link button is used frequently alongside the 'get emails' component, which both use the same styles (see [example page at the bottom](https://www.gov.uk/guidance/creative-content-exchange-research-privacy-notice)). And the button in the subscription links component sits alongside bold links, which does broadly match the button style there.

<img width="475" height="80" alt="Image" src="https://github.com/user-attachments/assets/07122bad-97d2-4bfa-ae59-e25eee7a88a1" />

We could also change the 'get emails' button to match whatever new button style we use, and change the whole look of the subscription links component, but it feels like we'd be changing an increasing number of buttons rather than acknowledging that this style is in common use on GOV.UK.

## Create a new common button style, and use that in both components
We could add another button style that looks like this into the button component, then just use that in all the places mentioned so far.

This would require handling the icons. We could modify the button component to yield to a block, and pass the icon and text, but there's a slight component isolation issue there where the component would need to style things passed to it (the SVG). The cleanest way to do it would be to have button variants with each of the icons built in (`printer`, `feed`, might as well include `bell`), but that does increase the number of button options - plus you'd need a bold text variant for the subscription links.

## Recommendation
Personally I think creating a new common button style in the button component and using that in other components is the best option, and one we can achieve without needing to redesign big chunks of the site. However it does increase the complexity of the button component and it would be a lot of work, and I'd question whether it is worth the effort.

**Comment by MartinJJones** [comment]
@andysellick nice investigation work and summary 🕵️ 

Looking at the remaining button issues:

[button in subscription links](https://components.publishing.service.gov.uk/component-guide/subscription_links/with_copyable_feed_link) | [print link button](https://components.publishing.service.gov.uk/component-guide/print_link)
---- | -----
<img width="415" height="81" alt="Image" src="https://github.com/user-attachments/assets/b1f491fb-5cad-4c37-a23d-145ede0703b7" /> | <img width="193" height="72" alt="Image" src="https://github.com/user-attachments/assets/8d67cab3-94f9-4a00-90c2-6eb0c8742d77" />

I'm wondering if the button can be removed altogether from the subscription links component, I searched alphagov for the `feed_link_box_value` option and cannot see this used anywhere - https://github.com/search?q=org%3Aalphagov+feed_link_box_value&type=code

If we are able to remove this option, it should avoid having to review the button styles further to get them to be consistent, as the subscription links component should then only include links.

There are likely further design improvements that can be made in the future to these components and others, updating icons for example, but I think this would be best handled separately.
